xbmc/pvr/windows/GUIWindowPVRBase.cpp

```diff
@@ -389,7 +389,20 @@ bool CGUIWindowPVRBase::OpenChannelGroupSelectionDialog(void)
 
 bool CGUIWindowPVRBase::InitChannelGroup()
 {
-  CPVRChannelGroupPtr group(CServiceBroker::GetPVRManager().GetPlayingGroup(m_bRadio));
+  std::shared_ptr<CPVRChannelGroup> group;
+  if (m_channelGroupPath.empty())
```

Comment:
Can the path really be empty?

Reply:
Yep, actually It will be in most use cases. Only if a parameter is passed to the channel or guide window with the on-window-init message it will ever be non-empty.  Look at the other changes I did in this PR and you will see.